types/find-node-modules/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for find-node-modules 2.1
+// Project: https://github.com/callumacrae/find-node-modules
+// Definitions by: Artem Klyuev <https://github.com/ArtemKlyuev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace findNodeModules {
+    interface Options {
+        /**
+         * The directory to start the search from
+         *
+         * @default process.cwd()
+         */
+        cwd?: string;
+        /**
+         * What directory to search for
+         *
+         * @default 'node_modules'
+         */
+        searchFor?: string;
+        /**
+         * If false, returns absolute paths
+         *
+         * @default true
+         */
+        relative?: boolean;
```

Comment:
You should add `| undefined` to optional types to support TypeScript's exact optional properties:

```suggestion
        cwd?: string | undefined;
        /**
         * What directory to search for
         *
         * @default 'node_modules'
         */
        searchFor?: string | undefined;
        /**
         * If false, returns absolute paths
         *
         * @default true
         */
        relative?: boolean | undefined;
```

Everything else LGTM!

Reply:
@MysteryBlokHed 
I added the code you suggested, please review again